pkgs/top-level/python-packages.nix

```diff
@@ -257,6 +257,10 @@ in {
     pythonPackages = self;
   };
 
+  pyqt56 = pkgs.libsForQt56.callPackage ../development/python-modules/pyqt/5.x.nix {
```

Comment:
Keep this private to leo-editor. We shouldn't have multiple versions of a package in `python-packages.nix`

Reply:
Okay, sure thing.  Sorry about that (pushed new commit with requested change).